.travis.yml

```diff
@@ -50,16 +57,18 @@ before_install:
 install:
     - pip install setuptools setuptools-scm
     - pip install -r requirements.txt
-    - git clone https://github.com/Statoil/libecl.git
+    - git clone --recursive https://github.com/Statoil/libecl.git
     - mkdir libecl/build
     - pushd libecl/build
+    - pip install -r ../requirements.txt
     - cmake .. -DBUILD_SHARED_LIBS=ON
+               -DPYTHON_EXECUTABLE=`which python`
```

Comment:
I don't think the `-DPYTHON_EXECUTABLE` switch to cmake or setting `PYTHONPATH` is required?

Reply:
It is for mac for some reason